applications/GeoMechanicsApplication/tests/test_helper.py

```diff
@@ -529,11 +530,32 @@ def _strip_off_quotes(self, quoted_string):
     @staticmethod
     def get_values_at_time(time, property_results):
         for time_results in property_results:
-            if time_results["time"] == time:
+            if math.isclose(time_results["time"], time):
                 return time_results["values"]
 
     @staticmethod
     def get_value_at_node(node, time_results):
         for node_results in time_results:
             if node_results["node"] == node:
                 return node_results["value"]
+
+    @staticmethod
+    def nodal_values_at_time(result_item_name, time, output_data, node_ids=None):
```

Comment:
It seems that this function has some overlap with the get_values_at_time and get_value_at_node. Do you think we could replace the older functions with this more elaborate one?

Reply:
Good point! Yes, I think we can. However, I would propose to do that in a separate PR.